src/apis/domains/users/queries.ts

```diff
@@ -0,0 +1,42 @@
+import { QueryClient, useMutation } from "@tanstack/react-query";
+import { postKakaoLogin } from "./api";
+import { useAtom } from "jotai";
+import { userAtom } from "@stores/user";
+import { components } from "@typings/api/schema";
+
+const QUERY_KEY = {
+  KAKAO_LOGIN: "kakaoLogin",
+};
+
+type LoginSuccessReponse = {
+  data: {
+    accessToken: string;
+    refreshToken: string;
+    nickName: string;
+  };
+};
```

Comment:
p4)
```suggestion
type LoginSuccessResponse = components["schemas"]["LoginSuccessResponse"];
```
이렇게 하면 타입이 맞게 하는데 리턴 타입에서 오류가 나네용 이따가 토큰 되면 확인해봐욥,,,